machine_learning/benchmarks/datasets.py

```diff
@@ -0,0 +1,81 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Dataset definitions for machine learning benchmarks."""
+
+import numpy as np
+from sklearn.datasets import load_iris
+
+DATASET_SYNTHETIC_FEATURES = np.array(
```

Comment:
Please be sure, names are consistent across both PRs.

Reply:
Changed to `DATASET_SYNTHETIC_CLASSIFICATION_FEATURES` which is now consistent with `DATASET_SYNTHETIC_REGRESSION_FEATURES`